src/components/graph/screens/GraphActivityDisplay/containers/GraphActivityDisplayContainer.js

```diff
@@ -19,46 +23,32 @@ class GraphActivityDisplayContainer extends React.Component {
     this.toARTNodes = this.toARTNodes.bind(this);
     this.toARTEdges = this.toARTEdges.bind(this);
     this.ticked = this.ticked.bind(this);
+    this.handleNodePress = this.handleNodePress.bind(this);
     this.reset = this.reset.bind(this);
     this.width = Dimensions.get('window').width;
     this.height = Dimensions.get('window').height;
-    this.data = [
-      { x: 0, y: 0, r: 50, label: 'jean', id: 1 },
-      { x: 200, y: 420, r: 50, label: 'marc', id: 2 },
-      { x: 200, y: 200, fx: this.width / 2, fy: this.height / 2, r: 50, label: 'john', id: 3 },
-      { x: 253, y: 126, r: 50, label: 'georges', id: 4 },
-      { x: 325, y: 463, r: 50, label: 'georges5', id: 5 },
-      { x: 320, y: 26, r: 50, label: 'georges6', id: 6 },
-      { x: 120, y: 256, r: 50, label: 'georges7', id: 7 },
-      { x: 420, y: 126, r: 50, label: 'georges8', id: 8 },
-    ];
-
-    this.links = [
-      { id: 1, source: this.data[1], target: this.data[2] },
-      { id: 2, source: this.data[0], target: this.data[2] },
-      { id: 3, source: this.data[3], target: this.data[2] },
-      { id: 4, source: this.data[4], target: this.data[2] },
-      { id: 5, source: this.data[5], target: this.data[2] },
-      { id: 6, source: this.data[6], target: this.data[2] },
-      { id: 7, source: this.data[7], target: this.data[2] },
-    ];
+    this.data = [];
+    this.links = [];
     this.state = {
       data: this.data,
-      links: this.links,
-      centralNodeId: this.props.centralNodeId,
     };
   }
 
   componentDidMount() {
-    this.force = d3.force.forceSimulation(this.data)
-      .force('charge', d3.force.forceManyBody(-300).distanceMax(this.height / 2))
-      .force('link', d3.force.forceLink(this.links).distance(200))
-      // .force('radial', d3.force.forceRadial(500, this.width / 2, this.height / 2))
-      // .force('center', d3.force.forceCenter(this.width / 2, this.height / 2))
-      // .force('x', d3.force.forceX().strength(0.2))
-      // .force('y', d3.force.forceY().strength(0.2))
-      .force('collide', d3.force.forceCollide().strength(1).radius(d => d.r))
-      .on('tick', this.ticked);
+    this.props.getAdjacentNodes(this.props.centerNodeId, GraphNodeType.TAG)
+      .then(() => {
+        this.data = this.props.nodesList;
+        this.links = this.props.linksList;
+        this.force = d3.force.forceSimulation(this.data)
+          .force('charge', d3.force.forceManyBody(-300).distanceMax(this.height / 2))
+          .force('link', d3.force.forceLink(this.links).distance(200))
+          // .force('radial', d3.force.forceRadial(500, this.width / 2, this.height / 2))
+          // .force('center', d3.force.forceCenter(this.width / 2, this.height / 2))
+          // .force('x', d3.force.forceX().strength(0.2))
```

Comment:
Do you still need those comments ?

Reply:
I think I will let them, then can be quite usefull if we want to try very fast new forces to make nodes intract